src/components/KDropdown/KDropdownItem.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <li
+    class="k-dropdown-item"
+    :class="{
+      'has-divider': hasDivider,
+      'disabled': disabled,
+      'danger': isDangerous,
+      'k-dropdown-selected-option': selected
+    }"
+    :data-testid="`k-dropdown-item-${label.replace(/ /gi, '-')}`"
```

Comment:
Yeah let's remove it. Some tests will break - that's a downside but this is a really poor practice.

Reply:
I think we need some sort of testid here though, right? maybe just `dropdown-item-{index}` or something? at least this way it's predictable as an integer instead of the label string

Reply:
Well now that the component test has failed - yeah I guess we do need a `testid` here 😅. Let's make it just `dropdown-item` since we can't know the `index` when it's slotted and I don't think we want to add a prop just for that. In tests we can still do `cy.getTestId('dropdown-item').eq(n)` to target the `n` element.